_config.yml

```diff
@@ -115,61 +115,38 @@ highlighter: rouge
 #     output: false
 #
 collections:
-  how-to-use-guide:
-    title: "How to use this guide"
-    description: "About this Content Guide and how you can use it."
-    order: 0
-    output: true
   content-structure:
     title: "Content structure"
     description: "The basic building blocks for crafting good content."
     order: 1
-    output: true
```

Comment:
Interesting this seems to have maybe fixed another little bug I was going to hopefully fix: the `Privacy statement` link was showing up in the `Content` column of the footer as well as under Feedback (the correct place). Yay!